src/Components/Sidebar/Sidebar.js

```diff
@@ -0,0 +1,58 @@
+import "./Sidebar.css";
+import { NavLink, useNavigate } from "react-router-dom";
+import { UserHandle } from "../UserHandle/UserHandle";
+const Sidebar = () => {
+  const activeStyle = {
+    borderBottom: "2px solid black",
+  };
+
+  const navigate = useNavigate();
+
+  return (
+    <div className="aside-section">
+      <div className="aside-section-fixed flex-col bord-3-purple justify-space-between-flex">
+        <div className="flex-col gap-2">
+          <div className="bord-3-red flex-row align-center-flex justify-center-flex">
+            <h3 className="curs-point" onClick={() => navigate("/")}>
+              BAT-NETWORK
+            </h3>
+          </div>
+          <div>
+            <ul className="flex-col gap-1 align-center-flex">
+              <li className="nav-item">
+                <NavLink
+                  to="/"
+                  style={({ isActive }) => (isActive ? activeStyle : undefined)}
```

Comment:
Instead of writing this piece of code for each NavLink, you can use .active class in your CSS file and define the style, that will apply to these links.